src/main/java/org/interledger/ilp/exceptions/InterledgerQuotingProtocolException.java

```diff
@@ -1,4 +1,4 @@
-package org.interledger.ilp.core.exceptions;
+package org.interledger.ilp.exceptions;
 
 public class InterledgerQuotingProtocolException extends Exception {
```

Comment:
Could do. The whole architecture of the Exceptions needs some review now that the overall architecture is stabilizing (kind of)